specs/core/0_beacon-chain.md

```diff
@@ -200,14 +200,15 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 | `EXITED_WITHOUT_PENALTY` | `3` |
 | `EXITED_WITH_PENALTY` | `4` |
 
-### Special record types
+### Max transactions
```

Comment:
Do we really want to call them "transactions"? I would be inclined to save that term for user-defined activity, and call them something more neutral, eg. "records".